content/en/docs/languages/java/configuration.md

```diff
@@ -152,8 +154,13 @@ Properties for configuring [resource](../sdk/#resource):
 [OS resource provider](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/instrumentation/resources/library/src/main/java/io/opentelemetry/instrumentation/resources/OsResourceProvider.java),
 set
 `-Dotel.java.disabled.resource.providers=io.opentelemetry.instrumentation.resources.OsResourceProvider`.
-See [ResourceProvider](#resourceprovider) for resource provider artifact
-coordinates.
+**NOTE**: The `otel.service.name` and `otel.resource.attributes` system
+properties / environment variables are interpreted in the
+`io.opentelemetry.sdk.autoconfigure.internal.EnvironmentResourceProvider`
```

Comment:
The `internal` package looks ugly here. Since we need to document the FQCN of resource providers, we should probably consider promoting this to a non-internal package.